blog/2021/02/julia-1.6-highlights.md

```diff
@@ -0,0 +1,389 @@
+@def title = "Julia 1.6 Highlights"
+@def authors = "Jeff Bezanson, Ian Butterworth, Nathan Daly, Keno Fischer, Tim Holy, Elliot Saba, Mosè Giordano, Stefan Karpinski"
+@def published = "28 February 2021"
+@def rss_pubdate = Date(2021, 2, 28)
+@def rss = """Some highlights of the Julia 1.6 release."""
+
+Julia version 1.6 has been released. Most Julia releases are timed and hence not
+planned around specific features, but this release was an exception since it is
+likely to become the next long-term support (LTS) release of Julia. Because of
+this we took extra time developing the release to make sure that features that
+are needed for the future health of the ecosystem made it into the release. The
+final decision about whether Julia 1.6 will become the new LTS will not be made
+until around the time that the 1.7 release enters stabilization, at which point
+we'll have a much better sense of how good of a release it was — like baked
+goods, some releases just come out better than others and you can't always tell
+until after they are baked and eaten. The full list of changes can be found in
+the [NEWS file](https://github.com/JuliaLang/julia/blob/release-1.6/NEWS.md),
+but here we'll give a more in depth overview of some of the release highlights.
+
+\toc
+
+## Parallel precompilation
+
+_Ian Butterworth_
+
+Executing all of the statements in a module often involves compiling a large
+amount of code, so Julia creates precompiled caches of the module to reduce this
+time. In 1.6, this package precompilation is faster and happens before you leave
+`pkg>` mode. Until now, precompilation took place solely as a single-processed
+sequence, precompiling dependencies one-by-one when needed during the linear
+code loading process when a package is `using`/`import`-ed for the first time.
+
+The olden days of <= 1.5:
+```julia-repl
+(v1.5) pkg> add DifferentialEquations
+...
+julia> @time using DifferentialEquations
+[ Info: Precompiling DifferentialEquations [0c46a032-eb83-5123-abaf-570d42b7fbaa]
+  474.288251 seconds …
+```
+In 1.6, `pkg>` mode gains a heavily parallelized precompile operation that is
+auto-invoked after package actions, to keep the active environment ready to
+load.
+```julia-repl
+(v1.6) pkg> add DifferentialEquations
+...
+Precompiling project...
+  Progress [========================================>]  112/112
+112 dependencies successfully precompiled in 72 seconds
+
+julia> @time using DifferentialEquations
+  4.995477 seconds …
+```
+Whereas the previous code loading precompilation process took ~8 minutes to
+precompile and load without indicating progress while doing so, the new
+mechanism took a little over 1 minute to precompile while showing progress through the
+dependencies, then the first time the package is loaded it happens at full
```

Comment:
Maybe my brain is melted at the moment, but the 2nd part of this sentence confuses me @IanButterworth "then the first time the package is loaded it happens at full speed."

What is happening at full speed? Perhaps a stand alone sentence here

Reply:
```suggestion
dependencies. Then the first time the package is loaded, it loads at full
```